packages/smooth_app/lib/helpers/text_input_formatters_helper.dart

```diff
@@ -107,10 +103,9 @@ class DecimalSeparatorRewriter extends TextInputFormatter {
   /// Replaces a "." by a "," or a "," by a "." only if
   /// the group separator is an empty character
```

Comment:
the comment about the group separator being an empty character should now be removed I guess